src/js/control-bar/time-controls/time-divider.js

```diff
@@ -20,7 +20,10 @@ class TimeDivider extends Component {
   createEl() {
     return super.createEl('div', {
       className: 'vjs-time-control vjs-time-divider',
-      innerHTML: '<div><span>/</span></div>'
+      // this element and its contents can be hidden from assistive techs since
+      // it is made extraneous by the announcement of the control text
+      // for the current time and duration displays
+      innerHTML: '<div aria-hidden="true"><span>/</span></div>'
```

Comment:
rather than it being added at this level, we should make the main time-divider div be aria-hidden, super.createEl may accent a 3rd argument object for properties, if not, you can use Dom.createEl directly and return that.